vagrant-spk

```diff
@@ -477,6 +477,19 @@ def auto(args):
 
 
 def setup_vm(args):
+    if (len(args.command_specific_args) == 0):
+        # No stack specified.  List known stacks.
+        print("No stack specified.  Try specifying a stack with:")
+        print("")
+        print("vagrant-spk setupvm <stack>")
+        print("")
+        print("Supported stacks:")
+        stacks_dir = os.path.join(CODE_DIR, "stacks")
+        for stack in sorted(os.listdir(stacks_dir)):
+            print("  {}".format(stack))
+        print("")
+        return
```

Comment:
In my opinon, this should `exit(1)` at the end. I'll merge as-is for latency-reduction reasons, but @zarvox if you're at least +0 on that, let me know and either you or I can make it `exit(1)`.